docs/modules/ROOT/pages/get-started.adoc

```diff
@@ -9,174 +9,168 @@ You need the following:
+Run a command that requires a connection to the cluster, such as `object list`:
+
++
+[source, clc]
+----
+CLC> \object list
+Connected to cluster: pr-3690
+
+------------------------------------
+ Service Name | Object Name
+------------------------------------
+ executor     | hot-backup-executor
+------------------------------------
+----
+
+[[step-3-cluster-switch]]
+== Step 3. Switch Between Clusters
+
 Having separate local configurations for your development and production clusters allows you to switch between them without leaving the command line. Make sure both clusters are running before you start.
 
-. From the command line, change into the `$HOME/.local/share/hz-cli` directory.
-. Execute the following command to use the default configuration to connect to your development cluster:
+. Execute the following command to use the `dev` configuration to connect to your development cluster:
 +
-```bash
-hzc
-```
+[source, bash]
+----
+clc -c dev
+----
+
 +
 As before, the Hazelcast CLC connects to your development cluster.
-. Type `exit` to return to `$HOME/.local/share/hz-cli`.
+. Type `\exit` to return to the command line.
 . Execute the following command to use the new configuration file for your production cluster.
 +
-```bash
-hzc -c <config_prod>.yaml
-```
+[source, bash]
+----
+clc -c prod
+----
 +
-The Hazelcast CLC connects to your production cluster.
+The Hazelcast CLC shell is started, using the configuration for the production cluster.
+
+[[step-4-write-data]]
+== Step 4. Write Data to a Map
 
-[[step-5-write-data]]
-== Step 5. Write Data to a Map
-Now that you've connected to both your clusters, try using the Hazelcast CLC to write data to a map on your development cluster. A quick and easy way to do this is to use a JSON file.
+Now that you've connected to both your clusters, try using the Hazelcast CLC to write data to a map on your development cluster.
+Let's write a script that adds some data to a map.
 
-. Copy the following JSON to a file in the `$HOME/.local/share/hz-cli` directory.
+. Create a file called `data.script` on your local machine and copy in the following lines.
 +
-.currencycodes.json
-```json
-{
-  "1": {
-        "Code": "CAD",
-        "Currency": "Canadian Dollar"
-      },
-  "2": {
-        "Code": "INR",
-        "Currency": "Indian Rupee"
-      },
-  "3": {
-        "Code": "MXN",
-        "Currency": "Mexican Peso"
-      },
-  "4": {
-        "Code": "GBP",
-        "Currency": "Pounds Sterling"
-      },
-  "5": {
-        "Code": "TRY",
-        "Currency": "Turkish Lira"
-      },
-  "6": {
-        "Code": "USD",
-        "Currency": "United States Dollar"
-      }
-}
-
-```
-. As before, from the command line change into the `$HOME/.local/share/hz-cli` directory. This time execute a `put-all` command to write the data from the JSON file to a new map called `currency`.
+.data.script
+[source]
+----
+\map set -n currency -k i32 1 -v json '{"Code": "CAD", "Currency": "Canadian Dollar"}'
```

Comment:
With `map set` I get the following error:

![image](https://user-images.githubusercontent.com/106963942/206233557-8a46388e-fcad-4a92-8259-5f115d1f8ef2.png)

With `map put` the data writes correctly, but see the following output:
![image](https://user-images.githubusercontent.com/106963942/206233886-90ce29c4-532a-4470-ae44-52ee8dfa60f6.png)

Reply:
I've removed `map put` in favor of `map set`. The change is available on master.